app/src/test/java/ru/vssemikoz/newsfeed/newsfeed/NewsFeedPresenterTest.java

```diff
@@ -30,6 +36,7 @@
 import static org.mockito.Mockito.when;
 
 @RunWith(MockitoJUnitRunner.class)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
Is it used to re-order test methods? We shouldn't rely on tests launch order. Please, debug your tests/code and find out the reason of the error.